Lecture4-Home-Work/src/com/company/Main.java

```diff
@@ -0,0 +1,68 @@
+package com.company;
+
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+        String data;
+        String filter = null;
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Please enter at least three words separated by commas.");
+        data = scanner.nextLine();
+
+        String[] words = data.split(",");
+
+        do {
+            System.out.println("Please choose a filter:"
+                    + "\n1 - maximum length"
+                    + "\n2 - start with"
+                    + "\n3 - end with"
+                    + "\n4 - contains"
+                    + "\n0 - exit");
+            filter = scanner.nextLine();
+            String searchParam;
+
+            if (filter.equals("1")) {
+                System.out.println("Please specify maximum length");
+                searchParam = scanner.nextLine();
+            } else if (filter.equals("2")) {
+                System.out.println("Please specify the first letter");
+                searchParam = scanner.nextLine();
+            } else if (filter.equals("3")) {
+                System.out.println("Please specify the last letter");
+                searchParam = scanner.nextLine();
+            } else if (filter.equals("4")) {
+                System.out.println("Please specify part of the word");
+                searchParam = scanner.nextLine();
+            } else if (filter.equals("0")) {
+                System.out.println("Exit");
+                break;
+            } else {
+                System.out.println("Please use only supported filters.");
+                continue;
+            }
+
+            System.out.println("Results: ");
+
+            for (String word : words) {
+                word = word.trim();
+
+                if (filter.equals("1") && word.length() <= Integer.parseInt(searchParam)) {
```

Comment:
please test ALL if statements and use correct operators

Reply:
fixed